tools/nl/embeddings/input/base/convert.py

```diff
@@ -0,0 +1,38 @@
+import csv
```

Comment:
nit: Should convert.py live elsewhere?

Reply:
Did not mean to check in this.. I will let it live for now and remove after converting all the csvs to yaml?

Reply:
I think its still useful to have because we have scripts that generate csvs for NL descriptions (eg for UN), so for future imports it could still be relevant.

Reply:
move this as a new tool now